app/helpers/StringHelper.php

```diff
@@ -84,24 +84,22 @@ function addUrls($string, $preview = false)
                             $content.
                             '</a>'
                         );
-                    } else {
-                        return stripslashes(
-                            '<a href=\"'.
-                            Route::urlize('contact', $link).
-                            '\">'.
-                            $content.
-                            '</a>'
-                        );
                     }
-                } elseif(filter_var($content, FILTER_VALIDATE_URL)) {
+                    return stripslashes(
+                        '<a href=\"'.
+                        Route::urlize('contact', $link).
+                        '\">'.
+                        $content.
+                        '</a>'
+                    );
+                }
+                if(filter_var($content, FILTER_VALIDATE_URL)) {
                     return stripslashes('<a href=\"'.$content.'\" target=\"_blank\" rel=\"noopener\">'.$content.'</a>').
                             ($lastTag !== false ? $lastTag : '');
-                } else {
```

Comment:
Those following changes has been tested?

Reply:
No, why would it potentially break?